label_studio/core/utils/contextlog.py

```diff
@@ -271,7 +271,7 @@ def create_payload(self, request, response, body):
         else:
             namespace = request.resolver_match.namespace if request.resolver_match else None
             status_code = response.status_code
-            content_type = response.content_type
+            content_type = response.content_type if 'content_type' in response else None
```

Comment:
does response support this check `'content_type' in response`? 

I would do something like this: 
```suggestion
            content_type = response.content_type if hasattr(response, 'content_type') else None
```

Reply:
Good point, I am not sure what methods the base classes have, better to use a more catch-all solution